File: server/helpers/enums/camera_url_status.py
from enum import Enum

class CameraUrlStatus(Enum):
    ACTIVE = 'ACTIVE'
    INACTIVE = 'INACTIVE'
    
    @classmethod
    def from_string(cls, urlStatus_str):
        for urlStatus in CameraUrlStatus:
            if urlStatus_str == urlStatus.name:
                return urlStatus
        return None
    
    def __str__(self) -> str:
        return self.name

File: client/lib/shared/enum/camera_url_status.dart
import 'package:freezed_annotation/freezed_annotation.dart';

enum CameraUrlStatus {
  @JsonValue("ACTIVE")
  active,
  @JsonValue("INACTIVE")
  inactive,
}

extension CameraUrlStatusExt on CameraUrlStatus {
  String get text => {
        // CameraUrlStatus.active: tr(LocaleKeys.aciv),
        // CameraUrlStatus.inactive: tr(LocaleKeys.CameraUrl_Inactive),
      }[this]!;
}
